brave/brave-core
# Transition Brave Ads database into library

Resolves https://github.com/brave/brave-browser/issues/10328

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm ad notifications work as expected
- Confirm ad conversions are persisted in `ads_service/database.sql`
- Confirm ad conversions expire in `ads_service/database.sql`(see `expiry_timestamp` field)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
